yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -273,56 +274,77 @@ func checkHasHeader() {
 	}
 }
 
-func checkAndParseFileOpts() {
+func checkAndParseEscapeAndQuoteChar() {
 	switch fileFormat {
 	case datafile.CSV:
-		// setting default values for escape and quote, will be updated if provided in fileOpts flag
-		fileOptsMap = map[string]string{
-			"escape_char": "\"",
-			"quote_char":  "\"",
+		// setting default values for escape and quote
+		if escapeChar == "" {
+			escapeChar = `"`
 		}
-		if strings.Trim(fileOpts, " ") == "" { // if fileOpts is empty, then return
-			return
+		if quoteChar == "" {
+			quoteChar = `"`
 		}
 
-		keyValuePairs := strings.Split(fileOpts, ",")
-		for _, keyValuePair := range keyValuePairs {
-			key, value := strings.Split(keyValuePair, "=")[0], strings.Split(keyValuePair, "=")[1]
-			key = strings.ToLower(key)
-			if !slices.Contains(supportedCsvFileOpts, key) {
-				utils.ErrExit("ERROR: %q is not a valid csv file option", key)
-			} else {
-				resolvedValue, ok := resolveAndCheckSingleByteChar(value)
-				if !ok {
-					utils.ErrExit("ERROR: invalid syntax of opt '%s=%s' in --file-opts flag. It should be a valid single-byte value.", key, value)
+		if fileOpts != "" {
+			keyValuePairs := strings.Split(fileOpts, ",")
+			for _, keyValuePair := range keyValuePairs {
+				key, value := strings.Split(keyValuePair, "=")[0], strings.Split(keyValuePair, "=")[1]
+				key = strings.ToLower(key)
+				if !slices.Contains(supportedCsvFileOpts, key) {
+					utils.ErrExit("ERROR: %q is not a valid csv file option", key)
+				} else {
+					if key == "escape_char" {
+						escapeChar = value
+					} else if key == "quote_char" {
+						quoteChar = value
+					}
 				}
-				fileOptsMap[key] = resolvedValue
 			}
 		}
 
+		resolvedEscapeChar, ok := resolveAndCheckSingleByteChar(escapeChar)
+		if !ok {
+			utils.ErrExit("ERROR: invalid syntax of --escape-char=%s flag. It should be a valid single-byte value.", escapeChar)
+		}
+		escapeChar = resolvedEscapeChar
+
+		resolvedQuoteChar, ok := resolveAndCheckSingleByteChar(quoteChar)
```

Comment:
Should we rename `resolveAndCheckSingleByteChar()` as `interpreteEscapeSequences()`?